src/main/java/in/projecteka/gateway/clients/ConsentRequestServiceClient.java

```diff
@@ -70,7 +70,20 @@
     }
 
     @Override
-    public Mono<Void> routeResponse(JsonNode request, String cmUrl) {
-        return null;
+    public Mono<Void> routeResponse(JsonNode request, String url) {
+        return centralRegistry.authenticate()
+                .flatMap(token -> Utils.serializeRequest(request)
+                        .flatMap(serializedRequest ->
+                                webClientBuilder.build()
+                                        .post()
+                                        .uri(url + "/v1/consent-requests/on-init")
+                                        .contentType(MediaType.APPLICATION_JSON)
+                                        .header(HttpHeaders.AUTHORIZATION, token)
+                                        .bodyValue(serializedRequest)
+                                        .retrieve()
+                                        .onStatus(httpStatus -> !httpStatus.is2xxSuccessful(),
+                                                clientResponse -> Mono.error(ClientError.unableToConnect()))//TODO Error handling
+                                        .bodyToMono(Void.class)
```

Comment:
you can use .toBodilessEntity to be more explicit.

Reply:
Done.